store/set.js

```diff
@@ -43,9 +43,5 @@ export const actions = {
   async createLikes({ commit }) {
     const likesId = await this.$sets.createLikes().then(response =>  response.id.split('/').pop());
     commit('setLikesId', likesId);
-  },
-  // TODO: is this used?
-  reset({ commit }) {
```

Comment:
Restore this as it is needed. You could remove the TODO note though.

Reply:
This is not needed anymore since the window is reloaded and store starts empty, it was added before to deal with auth.logout() that was not reloading the window.